bookwyrm/settings.py

```diff
@@ -310,13 +327,8 @@
     MEDIA_FULL_URL = MEDIA_URL
     STATIC_FULL_URL = STATIC_URL
     DEFAULT_FILE_STORAGE = "bookwyrm.storage_backends.ImagesStorage"
-    # I don't know if it's used, but the site crashes without it
```

Comment:
thank you for this

Reply:
It was handy, I was like "dang, I want these higher up in the file" and then realized that they were the same whether we were using S3 or not, so they could be! 😁